File: twoSum.js
function twoSum (nums, target) {
    let diff = 0;
    for (let i = 0; i < nums.length; i++) {
        diff = target - nums[i];
        console.log(diff);
        if(nums.includes(diff)) {
            return  [].concat(i, nums.indexOf(diff))
        }
    }
    
}

console.log(twoSum([2, 7, 15, 18], 22));

File: twoSum.py
class TwoSum: 
    def __init__(self):
        return None

    def two_sum(self, nums, target):
        diff = 0
        for num in nums:
            diff = target - num
            if diff in nums:
                return [nums.index(diff), nums.index(num)]

def main():
    solve = TwoSum()
    print(solve.two_sum([3,2,4], 6))

if __name__ == "__main__":
    main()